src/renderer/coremods/installer/commands.ts

```diff
@@ -0,0 +1,116 @@
+import { Injector, plugins } from "@replugged";
+import { ApplicationCommandOptionType } from "src/types";
+import { INSTALLER_SOURCES, InstallerSource, installFlow, installURL } from "./util";
+
+/**
+ * A map 'special' names that don't need to be translated by i18n
+ */
+const specialSourceDisplayNames: Partial<Record<InstallerSource, string>> = {
+  github: "GitHub",
+};
+
+function installSourceName(source: InstallerSource): string {
+  const displayName = specialSourceDisplayNames[source];
+  if (displayName) {
+    return displayName;
+  }
+
+  // TODO: i18n
+  return source;
+}
```

Comment:
I think this is way overcomplicated. Just have a display name map like you had before and hardcode the value for GH.

Reply:
The problem with that is translation. If theres a display name map, we can't use translations, right?

The other alternative is to remove the map in the first place, and only keep the installSourceName function. This way would be able to 'map' translation values, hard code some values like GitHub, and be easy to expand on

Reply:
Why wouldn't we be able to use translations? Anywhere we use a string to display text we should be able to display an i18n value instead.

Reply:
When are the translations initialized? If use this:

```ts
const sourceDisplayNames = {
   "store": Messages.*
}
```

Then wouldn't it be initialized with the wrong string?

In that case we could load it at start() but that might not be best practice?

Reply:
I think it wouldn't be an issue but you can test and see.

Reply:
Oh wow, you are actually correct. Thats odd I really didn't expect it, I'll update in a sec